stress/common/src/main/java/alluxio/stress/common/FileSystemParameters.java

```diff
@@ -33,9 +35,17 @@
       converter = FileSystemParameters.FileSystemParametersReadTypeConverter.class)
   public ReadType mReadType = ReadType.CACHE;
 
+  @Parameter(names = {"--write-type"},
+      description = "The write type to use when creating files. Options are [MUST_CACHE, "
+          + "CACHE_THROUGH, THROUGH, ASYNC_THROUGH, ALL]",
+      converter = FileSystemParameters.FileSystemParametersWriteTypeConverter.class)
+  public String mWriteType = InstancedConfiguration.defaults()
+      .get(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT);
+
```

Comment:
Isn't this duplicated in #14925 ?

Reply:
Yes, I am planning to remove the ALL option in the batchTaskRunner and make it a individual batchTask